### PR TITLE
feat(homepage): New homepage for docs

### PR DESCRIPTION
**Pull request by jbartadev**
closes https://github.com/apify/apify-web/issues/2244

### CODE REVIEW

File: src/components/CardWithIcon/CardWithIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from "react";
```
Comment (B4nan) [inline src/components/CardWithIcon/CardWithIcon.tsx:1]: i see mixed casing and indents, i guess we dont have the linter set up properly for tsx files. no need to deal with that manually.

**Review by B4nan — COMMENTED** [review]
the search box is a bit broken now, otherwise looking good, we can merge it and improve via later PRs if you want to get feedback from others (so they dont have to run this locally)

image

**Comment by barjin** [comment]
is the gradient background on the headings intentional? looks kinda off to me :)
image

**Comment by barjin** [comment]
the [search box](https://github.com/apify/apify-docs-v2/pull/14#pullrequestreview-1234746809) problem seems to be present in the sidebar as well, IMHO bigger problem here (the menu elements under the gradient are not clickable)
Peek 2023-01-04 08-39

**Review by nguyeda1 — APPROVED** [review]

**Pull request merged**